hera_mc/cm_hookup.py

```diff
@@ -38,14 +38,12 @@ def get_parts_from_hookup(part_name, hookup_dict):
     if part_name not in hookup_dict['columns']:
         return None
     parts = {}
-    part_ind = hookup_dict['columns'].index(part_name)
+    part_ind = hookup_dict['columns'].index(part_name) - 1 if hookup_dict['columns'].index(part_name) > 1 else 0
```

Comment:
Is this test correct? Right now we have that if `hookup_dict['columns'].index(part_name)` is 0 or 1, `part_ind = 0`. Otherwise, `part_ind = hookup_dict['columns'].index(part_name) - 1`. I don't understand the structure of the `columns` list and how that relates to the things in the `hookup` sub-structure, so maybe this is appropriate. If so some explanatory comments would help.

Reply:
It is but is confusing -- it matches the calls in the cm_info stuff.  I'd like to change both of these to be less confusing and more robust.  I'll make an issue.